README.md

```diff
@@ -185,6 +185,30 @@ const client = getClient();
 })();
 ```
 
+#### Primitives
+
+A high level goal of bPanel is to allow for many different blockchain
+backends. To allow for dynamic switching of chains when implementing
+logic with primitives, it is possible to use objects that are keyed
+by the blockchain name to the primitive. These objects are wrapped
+with some helper functions that work via the primitive `from` methods.
+Each of the exposed functions are named `toPrimitive` where primitive
+is a bcoin primitive.
```

Comment:
Nice. I like this description text a lot 👍

Reply:
👍